panics.go

```diff
@@ -72,8 +67,8 @@ func (w *writer) Write(p []byte) (int, error) {
 // uncaught panic. The Writer must be safe for concurrent use. By default,
 // panics are written to [os.Stderr].
 //
-// If writing the panic takes more that [WriteTimeout] all registered channels
-// and contexts will be notified / canceled before the write completes.
+// If writing the panic takes more than the timeout (configurable via the [WriteTimeout] function)
+// all registered channels and contexts will be notified / canceled before the write completes.
```

Comment:
nit, wrap lines at 80 chars (I'm kinda pedantic about this since I like to be able to view two panes of code side-by-side on my 14" Mac)